include/mbedtls/sha1.h

```diff
@@ -1,7 +1,10 @@
 /**
  * \file sha1.h
  *
- * \brief The SHA-1 cryptographic hash function.
+ * \brief This file contains SHA-1 definitions and functions.
+ *
+ * The Secure Hash Algorithm 1 (SHA-1) cryptographic hash function is defined in 
+ * <em>FIPS 180-4: Secure Hash Standard (SHS)</em>.
```

Comment:
Can we provide a link here, if we're referencing the standard?

Reply:
Usually documentation does not link to other documents that are not under its control, that is - a link may change, and you would not be aware, so your documentation would contain broken links. 
I can add links, if you wish, but you would need to assign someone to verify these before every release, which I think is redundant overhead. These standards are easy to find.